packages/kit/src/runtime/client/ambient.d.ts

```diff
@@ -1,6 +1,7 @@
-declare module '__GENERATED__/manifest.js' {
-	import { CSRComponent, CSRRoute } from 'types';
+declare module '__GENERATED__/client-manifest.js' {
+	import { CSRComponentLoader } from 'types';
 
-	export const fallback: [CSRComponent, CSRComponent];
-	export const routes: CSRRoute[];
+	export const components: CSRComponentLoader[];
+	export const dictionary: Record<string, [number[], number[], 1]>;
```

Comment:
it could be worth a JSDoc explaining what dictionary is. it wasn't really clear to me from the name alone. I see that it's value is set to `routes`. I wonder if just calling it `routes` might be a more informative name?

Reply:
added some docs